.changeset/witty-avocados-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+[breaking] remove Prerendered.assets
```

Comment:
this doesn't reflect what's happening in this PR — was this a separate piece of work we were hoping to include here?

Reply:
No. That should be updated. It accidentally got included from another PR I was considering sending

Reply:
No objections assuming it's not used anywhere. It was an internal-only thing, so we shouldn't worry about anyone else relying on it

Reply:
```suggestion
prerender in a subprocess
```